front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/index.tsx

```diff
@@ -1,491 +1,99 @@
+            }}
           />
-        </SectionField>
-        {type === 'homePage' && (
-          <>
-            <SectionField>
-              <InputMultilocWithLocaleSwitcher
-                type="text"
-                valueMultiloc={localSettings.banner_signed_in_header_multiloc}
-                label={formatMessage(messages.bannerHeaderSignedIn)}
-                onChange={handleSignedInHeaderOnChange}
-              />
-            </SectionField>
-            <SectionField
-              key="avatars"
-              data-cy="e2e-banner-avatar-toggle-section"
-            >
-              <SubSectionTitle>
-                <FormattedMessage {...messages.avatarsTitle} />
-              </SubSectionTitle>
-              <Setting>
-                <ToggleLabel>
-                  <StyledToggle
-                    checked={!!localSettings.banner_avatars_enabled}
-                    onChange={() => {
-                      updateValueInLocalState(
-                        'banner_avatars_enabled',
-                        !localSettings.banner_avatars_enabled
-                      );
-                    }}
-                  />
-                  <LabelContent>
-                    <LabelTitle>
-                      {formatMessage(messages.bannerDisplayHeaderAvatars)}
-                    </LabelTitle>
-                    <LabelDescription>
-                      {formatMessage(
-                        messages.bannerDisplayHeaderAvatarsSubtitle
-                      )}
-                    </LabelDescription>
-                  </LabelContent>
-                </ToggleLabel>
-              </Setting>
-            </SectionField>
-          </>
-        )}
-        {/* // CTA settings, only applicable on homepage */}
-        <Outlet
-          id="app.containers.Admin.settings.customize.headerSectionEnd"
-          banner_cta_signed_in_text_multiloc={
-            localSettings.banner_cta_signed_in_text_multiloc
-          }
-          banner_cta_signed_in_url={localSettings.banner_cta_signed_in_url}
-          banner_cta_signed_in_type={localSettings.banner_cta_signed_in_type}
-          banner_cta_signed_out_text_multiloc={
-            localSettings.banner_cta_signed_out_text_multiloc
-          }
-          banner_cta_signed_out_url={localSettings.banner_cta_signed_out_url}
-          banner_cta_signed_out_type={localSettings.banner_cta_signed_out_type}
-          handleOnChange={updateValueInLocalState}
-          // signed in only applies to
-          showSignedInSettings={type === 'homePage'}
-          errors={apiErrors}
-        />
-      </Section>
-    </SectionFormWrapper>
+        }
+      >
+        <Section key={'header'}>
+          {/*
+            Padding equal to the height of the sticky bar with the submit button.
+          */}
+          <Box pb="78px">
+            <Warning>
+              <FormattedMessage {...messages.heroBannerInfoBar} />
+            </Warning>
+            {outletSectionStart}
```

Comment:
I'm not sure about this entire wrapper. Not sure it makes sense to change it now, but the wrapper basically does nothing except have the section wrapper and then render various children if/when passed in to the form component. This makes the props for the form component huge. An alternative would be ditching this component and just importing the various sub-components in the two wrapper files and laying them out as a normal React component (this is effectively already happening, but with this form component in the middle).